ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -170,70 +170,81 @@ static int ompi_osc_rdma_pvar_read (const struct mca_base_pvar_t *pvar, void *va
 
 static int ompi_osc_rdma_component_register (void)
 {
+    char *description_str;
     mca_osc_rdma_component.no_locks = false;
+    asprintf(&description_str, "%s%s%s%s", "Enable optimizations available only if MPI_LOCK is ",
```

Comment:
Minor style thing -- you really only need one `%s`, even if your string constant spills over multiple lines.  Perhaps something like this:

```c
asprintf(&description_str, "Enable optimizations available only if MPI_LOCK is "
    "not used. Info key of same name overrides this value (default: %s)",
    (mca_osc_rdma_component.no_locks ? "true" : "false"));
```